delivery/components/DecisioningMode.yaml

```diff
@@ -0,0 +1,4 @@
+openapi: "3.0.0"
+DecisioningMode:
+  type: string
+  enum: ["server-side", "on-device", "on-device-cached"]
```

Comment:
Update with "edge", "local"

Reply:
@dcottingham @andyruwruw It should actually be removed, since we already have ExecutionMode and DecisioningMethod